packages/cli/src/entrypoint.ts

```diff
@@ -4,11 +4,22 @@ import { Package } from "./package";
 import { JSONValue } from "./utils";
 import normalizePath from "normalize-path";
 
+export type ExportsConditions = {
+  module?: string;
+  default?: string;
+};
+
+export type ExportsItem = {
+  browser?: ExportsConditions | string;
+  worker?: ExportsConditions | string;
+} & ExportsConditions;
+
 export class Entrypoint extends Item<{
   main?: JSONValue;
   module?: JSONValue;
   "umd:main"?: JSONValue;
   browser?: JSONValue;
+  exports?: Record<string, ExportsItem | string>;
```

Comment:
q: should we enforce `package.json#type` somehow if `package.json#exports` is there?